src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: $secondary;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
```

Comment:
it is a common practice to apply box-sizing via `*` selector since it is not explicitly when approach to calculate size is different for different elements